src/4.8/en/textui.xml

```diff
@@ -647,6 +647,39 @@ class TestCaseClass extends \PHPUnit_Framework_TestCase
             Use colors in output.
             On Windows, use <ulink url="https://github.com/adoxa/ansicon">ANSICON</ulink> or <ulink url="https://github.com/Maximus5/ConEmu">ConEmu</ulink>.
           </para>
+          <para>
+            There are three possible values for this option:
+            <itemizedlist>
+              <listitem>
+                <para>
+                  <literal>never</literal>: never displays colors in the output. This is the default value when <literal>--colors</literal> option is not used.
+                </para>
+              </listitem>
+              <listitem>
+                <para>
+                  <literal>auto</literal>: displays colors in the output unless the current terminal doesn't supports colors,
+                  or if the output is piped to a command or redirected to a file.
+                </para>
+              </listitem>
+              <listitem>
+                <para>
+                  <literal>always</literal>: always displays colors in the output even when the current terminal doesn't supports colors,
```

Comment:
There is a typo (my fault) on this and the other 2 topics above:

> current terminal doesn't supports colors

It must be _current terminal doesn't support colors_.